ProcessLib/PhaseField/PhaseFieldFEM.h

```diff
@@ -57,6 +57,7 @@ struct IntegrationPointData final
     double integration_weight;
     double history_variable;
     double history_variable_prev;
+    double pressure = 0.1;
```

Comment:
0.1 is a magic number; please add a comment.

Reply:
I changed it to unity (=1.0) and added explanations why.

Reply:
Explanations have been added...

Reply:
It's been changed to 1.0 and expiations why have been added.